examples/server.js

```diff
@@ -10,11 +10,11 @@ function listDirs(root) {
   var files = fs.readdirSync(root);
   var dirs = [];
 
-  for (var i=0, l=files.length; i<l; i++) {
+  for (var i=0, l=files.length; i<l; i++){
```

Comment:
format: style mismatch:
```suggestion
  for(var i = 0, l = files.length; i <l; i++) {
```

`8418e2cd` ␣ at column 6 should be removed.
`4e4e06c5` ␣ should be inserted at column 13.
`04ea5c96` ␣ should be inserted at column 14.
`4e4e06c5` ␣ should be inserted at column 18.
`04ea5c96` ␣ should be inserted at column 19.
`2f9d4b4e` ␣ should be inserted at column 34.
`39615007` ␣ should be inserted at column 42.